test/util/upgrade.js

```diff
@@ -163,9 +163,22 @@ async function upgradeClients(tag) {
     shell.exec(`git checkout HEAD contracts`);
   }
 
-  // Upgrade clients
   await hre.run("compile");
-  await hre.run("upgrade-clients", { env: "upgrade-test" });
+  // Mock forwarder to test metatx
+  const MockNativeMetaTransaction = await ethers.getContractFactory("MockNativeMetaTransaction");
+
+  const forwarder = await MockNativeMetaTransaction.deploy("Forwarder", "0.0.1");
```

Comment:
Here we deploy the forwarder when upgrading the client.
However I didn't see anything in the deploy-suite to deploy the forwarder in a fresh installation. Did I miss it?

Reply:
We haven't decided yet who will be the forwarder,  I guess we want to use biconomy?

Reply:
OK
However we will have to deploy our own forwarder in our local hardhat environment (as no Biconomy there).
I assume we can do it at the core-component level (our own deployment script), so that there is no need to add it into the deploy-suite task.